roles/installer/defaults/main.yml

```
@@ -237,8 +237,8 @@ extra_volumes: ''
 
 _image: quay.io/ansible/awx
 _image_version: "{{ lookup('env', 'DEFAULT_AWX_VERSION') or 'latest' }}"
-_redis_image: docker.io/redis
-_redis_image_version: 7
+_redis_image: quay.io/sclorg/redis-6-c8s
```

```suggestion
_redis_image: quay.io/sclorg/redis-6-c9s
```
Since the postgres 15 PR will use a c9s based container image then we should have the same thing everywhere.
What do you think ?

---

Agreed, I just changed this to c9s.  I'll make the same changes for eda-server-operator.